File: Assetmanagement/Assetmanagement/Models/RaisePOModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Assetmanagement.Models
{
    public class RaisePOModel
    {
        public long AssetQuotationId { get; set; }
        public long AssetRequestId { get; set; }
        public long AssetTypeId { get; set; }
        public string AssetTypeName { get; set; }
        public string VendorName { get; set; }
        public string AssetStatusName { get; set; }
        public long ?ApprovedBy { get; set; }
        public DateTime ?ApprovedOn { get; set; }
        public string ARNumber { get; set; }
        public string EmployeeName { get; set; }
        public string PONumber { get; set; }
        public string Location { get; set; }
    }
}

File: AMS_Source/src/app/views/assetrequests/raisepos/shared/raisepo.model.ts

export class ApproveAssetQuotationRaisePO {
    AssetQuotationId: string;
    AssetRequestId: string;
    AssetTypeId: string;
    AssetTypeName: string;
    VendorName: string;
    AssetStatusName: string;
    ApprovedBy: string;
    ApprovedOn: string;
    EmployeeName: string;
    ARNumber: string;
  
}

export class PaymentMode {
    constructor(public id: number, public name: string) { }
  }


export class PaymentDetailss{
    Cheque: string;
    CC: string;
    ChequeNo: string;
    ChequeAmount: string;
    ChequeDate:string;
}

export class ViewRaisePOModel{
    PONumber: string;
    AssetTypeName: string;
    Description: string;
    Quantity: string;
    Price: string;
    CGST: string;
    SGST: string;
    TotalPrice: string;
    ChequeNumber:string;
    CreditcardNumber:string;
    TransactionID:string;
}

export class ARNumber{
    ARNumber: string;
}
export class PaymentDetails{
    PaymentMode:string;
    BaseAmount: string;
    Tax: string;
    PaymentId: string;
    ARNumber: string;
    AssetType: string;
    RequestedBy: string;
    CreatedOn: string;
}

export class OtherAssetTypeSpecification{
    AssetTypeOtherpecification: string;
}
